views/score_multiple_words.erb

```diff
@@ -0,0 +1,27 @@
+<form action="" method="post" accept-charset="utf-8">
+  <label for="number">How many words do you want to score?</label>
+  <select name="number">
+    <% 100.times do |count| %>
+      <%= "<option value='#{ count + 1 }'>#{ count + 1 }</option>" %>
+    <% end %>
+  </select>
+
+  <input type="submit" value="This Many">
+</form>
+
+<% if @no_words %>
+  <form action="" method="post" accept-charset="utf-8">
+    <% @no_words.times do |count| %>
+      <p><label>Word <%= "#{ count + 1 }" %>: <input type="text" name="words[]"></label></p>
+    <% end %>
+    <input type="submit" value="Score Words">
+  </form>
+<% end %>
+
+<% if @scores %>
+  <% @scores.each do |word_score| %>
+    <% @word_score = word_score %>
+    <p><%= erb :display_word %></p>
```

Comment:
Since you're surrounding this in `<p>` both here and in the score_word views, it would make sense to actually put the `<p>` in the display_word view rather than repeat it both here and in score_word.